tests/test_slogpy_file_info.py

```diff
@@ -0,0 +1,59 @@
+"""UnitTesting for slog.info to file"""
+import os
+import re
+import tempfile
+
+import pytest
+
+from slogpy.slog import Slog as slog  # noqa: N813
+
+
+@pytest.fixture(scope="function")
+def temp_log_file():
+    """Fixture to initialize the logger with file logging enabled and cleanup afterwards"""
```

Comment:
Praise for coming up with a creative way of dealing with cleanup.  The thing is this is more creative than most people will understand quickly.  This means it is probably a good idea to explain what you are doing.  There is more than one bit of "magic" involved here.

1. Pytest fixtures
2. Scope for said fixture
3. Use tempfile to create a directory (instead of a file) so you can have it do the cleanup
4. Use of with to get tempfile to delete the directory that it creates
5. Use of yield to return value
6. Use of code after the yield to do the clean up, because pytest will do clean up and that will cause the code after yield to run

Reply:
Thank you for the praise and i will add the comments to explain this better in next PR